bgcval2/analysis_compare.py

```diff
@@ -409,6 +413,25 @@ def timeseries_compare(jobs,
             units = av[name]['modeldetails']['units']
 
             ts = 'Together'
+
+            if savecsv:
+                tst.save_csv(
+                        timesD,
+                        arrD, 
+                        analysisname,
+                        colours=colours,
+                        linestyles=linestyles,
+                        thicknesses=lineThicknesses,
+                        labels=labels,
+                        name=name,
+                        units=units,
+                        region=region,
+                        layer=layer,
+                        metric=metric,
+                        title=title,
+                        ts=ts,
+                        csvFolder=csvFolder)
+                    
```

Comment:
gets the prize for the function with most number of args I've seen :grin:

Reply:
![image](https://github.com/valeriupredoi/bgcval2/assets/4180685/6413193d-527d-42c2-9123-3a0185fd9dd9)